#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_profiler-support.dpatch by Norbert Tretkowski <nobse@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Tweak message about profiler support, we _are_ using Debian :)

@DPATCH@
diff -Nur ipython-0.7.0.orig/IPython/Magic.py ipython-0.7.0/IPython/Magic.py
--- ipython-0.7.0.orig/IPython/Magic.py	2006-01-10 08:06:14.000000000 +0000
+++ ipython-0.7.0/IPython/Magic.py	2006-01-21 19:31:53.000000000 +0000
@@ -90,9 +90,9 @@
 
     def profile_missing_notice(self, *args, **kwargs):
         error("""\
-The profile module could not be found.  If you are a Debian user,
-it has been removed from the standard Debian package because of its non-free
-license. To use profiling, please install"python2.3-profiler" from non-free.""")
+The profile module could not be found. It has been removed from the standard
+python packages because of its non-free license. To use profiling, install the
+python-profiler package from non-free.""")
 
     def default_option(self,fn,optstr):
         """Make an entry in the options_table for fn, with value optstr"""
